src/pres/loader/PGNLoader.js

```diff
@@ -15,37 +15,33 @@ export default class PGNLoader extends React.Component {
 
     constructor(props) {
         super(props)
-        let selectedSite = new URLSearchParams(window.location.search).get("source")
 
+        const urlSettings = this.props.urlSettings
         this.state = {
-            playerName: '',
-            site: selectedSite?selectedSite:'',
-            playerColor: this.props.settings.playerColor,
-            isAdvancedFiltersOpen: false,
+            playerName: urlSettings.playerName,
+            site: urlSettings.site,
+            playerColor: urlSettings.color || this.props.settings.playerColor,
```

Comment:
This is the one place (so far) where `urlSettings` and `settings` collide, by the way. I'm not sure what the correct behavior should be (I'm not really sure what `settings.playerColor` is, actually).

See also: urlSettings vs settings https://github.com/openingtree/openingtree/pull/273/files#r594764050